### PR TITLE
Fix restore focus

### PR DESCRIPTION
**Pull request by rtrompier**

### CODE REVIEW

**Comment by XciD** [comment]
nest +1

File: dist/README.md [hunk 1]
```diff
@@ -101,6 +101,9 @@ Queues a callback function to be called when the block has finished. This can be
 * **callback** (function)
 The callback function to queue.
 
+#### isBlocking
```
Comment (pascalberger) [inline dist/README.md:104]: isBlocking is already merged with #65 and should not be part of this PR

File: dist/angular-block-ui.js [hunk 2]
```diff
@@ -458,6 +458,10 @@ blkUI.factory('blockUI', ["blockUIConfig", "$timeout", "blockUIUtils", "$documen
       }
     };
 
+    this.isBlocking = function () {
```
Comment (pascalberger) [inline dist/angular-block-ui.js:461]: isBlocking is already merged with #65 and should not be part of this PR

**Comment by pascalberger** [comment]
Is this PR related to #61?

File: dist/angular-block-ui.css [hunk 3]
```diff
@@ -81,15 +81,13 @@ body.block-ui > .block-ui-container,
 /* - - - - - - 8-< - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - */
 
 .block-ui-anim-fade > .block-ui-container {
-  -webkit-transition: height 0s linear 200ms, opacity 200ms ease 0s;
-          transition: height 0s linear 200ms, opacity 200ms ease 0s;
+  transition: height 0s linear 200ms, opacity 200ms ease 0s;
 }
```
Comment (pascalberger) [inline dist/angular-block-ui.css:85]: I don't see how this change can be related to the focus problem

File: dist/angular-block-ui.css [hunk 4]
```diff
@@ -81,15 +81,13 @@ body.block-ui > .block-ui-container,
...
 
 .block-ui-anim-fade.block-ui-active > .block-ui-container {
   /*this resets the initial delay of the height */
   /*and sizes the block to full height at once at the start of the block. */
-  -webkit-transition-delay: 0s;
-          transition-delay: 0s;
+  transition-delay: 0s;
 }
```
Comment (pascalberger) [inline dist/angular-block-ui.css:91]: I don't see how this change can be related to the focus problem
